src/modules/fw_pos_control_l1/fw_pos_control_l1_params.c

```diff
@@ -98,7 +98,7 @@ PARAM_DEFINE_FLOAT(FW_THR_CRUISE, 0.6f);
  * The default value of 0 will disable scaling.
  *
  * @min 0.0
- * @max 2.0
+ * @max 10.0
```

Comment:
Why 10? Even 2 was questionable.

Reply:
As this changes the scale to be applied only on the amount of change the factor required is significantly higher. In our case approximately 3 is required